src/app/views/feelings/room/room.component.css

```diff
@@ -1,5 +1,5 @@
-.room{
-    padding-top: 60px;
+.room {
+    padding-top: 8rem;
```

Comment:
Read comment of the last file (i'm going from bottom to top because of the documentation files